bookwyrm/views/status.py

```diff
@@ -201,12 +201,11 @@ def edit_readthrough(request):
     # TODO: remove this, it duplicates the code in the ReadThrough view
     readthrough = get_object_or_404(models.ReadThrough, id=request.POST.get("id"))
 
-    readthrough.start_date = load_date_in_user_tz_as_utc(
-        request.POST.get("start_date"), request.user
-    )
-    readthrough.finish_date = load_date_in_user_tz_as_utc(
-        request.POST.get("finish_date"), request.user
-    )
+    if start_date := request.POST.get("start_date"):
+        readthrough.start_date = load_date_in_user_tz_as_utc(start_date, request.user)
```

Comment:
Another option would be to ensure the Comment form has the ReadThrough's start_date (as with the "Update progress" modal). I don't have a clear idea if it'd be a better approach here.

Reply:
I don't know either, which to me makes your solution seem equally good :)